requirements/pip.txt

```diff
@@ -83,7 +83,7 @@ git+https://github.com/mozilla/unicode-slugify@b696c37#egg=unicode-slugify==0.1.
 django-formtools==2.2
 django-crispy-forms==1.9.2
 
-docker==4.2.2
+docker==4.3.1
```

Comment:
Important changes here are:

- Set default API version to auto
- Update default API version to v1.39
- Update test engine version to 19.03.12

https://github.com/docker/docker-py/blob/4.3.1-release/docs/change-log.md